clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -153,6 +155,11 @@ class AArch64TargetCodeGenInfo : public TargetCodeGenInfo {
     }
     return TargetCodeGenInfo::isScalarizableAsmOperand(CGF, Ty);
   }
+
+  void checkFunctionCallABI(CodeGenModule &CGM, SourceLocation CallLoc,
```

Comment:
where is this new function called?

Reply:
It's called from CodeGenFunction::EmitCall in CGCall.cpp. x86 implements this function as well.